File: app/src/main/java/com/easy/lokalsolution/Activity/UploadFromHareActivity.java
package com.easy.lokalsolution.Activity;

import androidx.appcompat.app.AppCompatActivity;

import android.content.Intent;
import android.os.Bundle;
import android.view.View;

import com.easy.lokalsolution.databinding.ActivityUploadFromHareBinding;

public class UploadFromHareActivity extends AppCompatActivity {

    ActivityUploadFromHareBinding binding;

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        binding = ActivityUploadFromHareBinding.inflate(getLayoutInflater());
        setContentView(binding.getRoot());

        binding.addnews.setOnClickListener(new View.OnClickListener() {
            @Override
            public void onClick(View view) {
                Intent intent = new Intent(UploadFromHareActivity.this, AddNewsActivity.class);
                startActivity(intent);
            }
        });
        binding.addquery.setOnClickListener(new View.OnClickListener() {
            @Override
            public void onClick(View view) {
                Intent intent = new Intent(UploadFromHareActivity.this, AddQueryActivity.class);
                startActivity(intent);
            }
        });
        binding.createid.setOnClickListener(new View.OnClickListener() {
            @Override
            public void onClick(View view) {
                Intent intent = new Intent(UploadFromHareActivity.this, AddIdActivity.class);
                startActivity(intent);
            }
        });
        binding.back.setOnClickListener(new View.OnClickListener() {
            @Override
            public void onClick(View view) {
               finish();
            }
        });


    }
}

File: app/src/main/java/com/easy/lokalsolution/Activity/UploadFromHareActivity.kt
package com.easy.lokalsolution.Activity

import android.content.Intent
import android.os.Bundle
import androidx.appcompat.app.AppCompatActivity
import com.easy.lokalsolution.databinding.ActivityUploadFromHareBinding

class UploadFromHareActivity() : AppCompatActivity() {
    var binding: ActivityUploadFromHareBinding? = null
    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        binding = ActivityUploadFromHareBinding.inflate(layoutInflater)
        setContentView(binding!!.root)
        binding!!.addnews.setOnClickListener {
            val intent: Intent =
                Intent(this@UploadFromHareActivity, AddNewsActivity::class.java)
            startActivity(intent)
        }
        binding!!.addquery.setOnClickListener {
            val intent: Intent =
                Intent(this@UploadFromHareActivity, AddQueryActivity::class.java)
            startActivity(intent)
        }
        binding!!.createid.setOnClickListener {
            val intent: Intent = Intent(this@UploadFromHareActivity, AddIdActivity::class.java)
            startActivity(intent)
        }
        binding!!.back.setOnClickListener { finish() }
    }
}